components/base/kinematicbase/ptgKinematics.go

```diff
@@ -176,24 +209,15 @@ func (ptgk *ptgBaseKinematics) ErrorState(ctx context.Context) (spatialmath.Pose
 
 	// Determine the nominal pose, that is, the pose where the robot ought be if it had followed the plan perfectly up until this point.
 	ptgk.inputLock.RLock()
-	currentIdx := ptgk.currentIdx
-	currentExecutingSteps := ptgk.currentExecutingSteps
+	currentIdx := ptgk.currentState.currentIdx
+	currentExecutingSteps := ptgk.currentState.currentExecutingSteps
 	ptgk.inputLock.RUnlock()
 	currentInputs, err := ptgk.CurrentInputs(ctx)
```

Comment:
It seems wrong to me that our `currentState` has `currentInputs` information which we are choosing to not use here in favor of explicitly calling `ptgk.CurrentInputs(ctx)`. Are we worries about timing here?

Reply:
Good point, will move